frontends/mrxl/test/sum.mrxl

```diff
@@ -1,3 +1,3 @@
 input foo: int[4]
-output baz: int
-baz := reduce 5 (a, b <- foo) 0 { a + b }
+output baz: int[4]
```

Comment:
Same question as above, I suppose: it would seem to me like `reduce` should produce a scalar.

Reply:
That's the only way to observe its output in Calyx since marking registers with `@external` doesn't do anything

Reply:
Ah, got it, makes sense. Not something that we need to address now, but it would be pretty chill if the MrXL code could use a scalar but we could emit it (behind the scenes) as a 1-element memory.